services/main-frontend/src/pages/manage/exercise-services.tsx

```diff
@@ -165,6 +204,13 @@ const ExerciseServiceCard: React.FC<ExerciseServiceCardProps> = ({
   }
 
   const updateContent = async () => {
+    if (!canSave(service)) {
+      setStatus("failed")
+      window.setTimeout(() => {
+        setStatus(null)
```

Comment:
Not in a timeout, do as the first thing in updateContent so that the error resets on the next change.